src/app.py

```diff
@@ -1,13 +1,25 @@
 import time
+import eel
+import moisture_reading
 from controller import Controller
 from plant import Plant
```

Comment:
Does the app need to import plant if moisture reading is being imported and already imports plant?

Reply:
Good point